.github/workflows/ci.yml

```diff
@@ -163,3 +163,23 @@ jobs:
       with:
         name: ${{ matrix.browser-channel }}-${{ matrix.os }}
         path: pw-log.txt
+
+  build-conda:
```

Comment:
maybe we can only run it when the setup.py is changed? Since it contains the playwright driver version and that would avoid us having 3 extra bots for each run.

Reply:
Better to keep it for now since conda build is not as same as pip, remember that setuptools find_package one which was different from pip, and hence we would be safe from potential breakage